packages/twenty-front/src/pages/settings/security/SettingsSecurityTrustedDomain.tsx

```diff
@@ -0,0 +1,141 @@
+
+  const formConfig = useForm<{ domain: string; email: string }>({
+    mode: 'onChange',
+    resolver: zodResolver(
+      z
+        .object({
+          domain: domainSchema,
+          email: z.string().min(1),
+        })
+        .strict(),
+    ),
+    defaultValues: {
+      email: '',
+      domain: '',
+    },
+  });
+
+  const domain = formConfig.watch('domain');
+
+  const handleSave = async () => {
+    try {
+      createWorkspaceTrustDomain({
+        variables: {
+          input: {
+            domain: formConfig.getValues('domain'),
+            email:
+              formConfig.getValues('email') +
+              '@' +
+              formConfig.getValues('domain'),
+          },
+        },
+        onCompleted: () => {
+          enqueueSnackBar(t`Domain added successfully.`, {
+            variant: SnackBarVariant.Success,
+          });
+          navigate(SettingsPath.Security);
+        },
+        onError: (error) => {
+          enqueueSnackBar((error as Error).message, {
+            variant: SnackBarVariant.Error,
+          });
+        },
+      });
+    } catch (error) {
+      enqueueSnackBar((error as Error).message, {
+        variant: SnackBarVariant.Error,
+      });
+    }
+  };
+
+  return (
+    <SubMenuTopBarContainer
+      title="New Trusted Email Domain"
+      actionButton={
+        <SaveAndCancelButtons
+          isSaveDisabled={!formConfig.formState.isValid}
+          onCancel={() => navigate(SettingsPath.Security)}
+          onSave={handleSave}
+        />
+      }
+      links={[
+        {
+          children: <Trans>Workspace</Trans>,
+          href: getSettingsPath(SettingsPath.Workspace),
+        },
+        {
+          children: <Trans>Security</Trans>,
+          href: getSettingsPath(SettingsPath.Security),
+        },
+        { children: <Trans>New Trusted Domain</Trans> },
+      ]}
+    >
+      <SettingsPageContainer>
+        <Section>
+          <H2Title title="Domain" description="The name of your Domain" />
+          <Controller
+            name="domain"
+            control={formConfig.control}
+            render={({ field: { onChange, value } }) => (
+              <TextInput
+                autoComplete="off"
+                label="Domain"
+                value={value}
+                onChange={(domain: string) => {
+                  onChange(domain);
+                }}
+                fullWidth
+                placeholder="yourdomain.com"
+              />
+            )}
+          />
+        </Section>
+        <Section>
+          <H2Title
+            title="Email verification"
+            description="We will send your a link to verify domain ownership"
+          />
+          <Controller
+            name="email"
+            control={formConfig.control}
+            render={({ field: { onChange, value } }) => (
+              <TextInput
+                autoComplete="off"
+                label="Email"
```

Comment:
When there is just one item in the section then there is no input label, for example:
<img width="557" alt="Screenshot 2025-02-20 at 09 13 15" src="https://github.com/user-attachments/assets/30be7b1a-5a1f-4a9f-9d9e-ab3f7e422417" />